packages/paste-website/src/pages/components/callout/index.mdx

```diff
@@ -0,0 +1,236 @@
+---
+title: Callout
+package: '@twilio-paste/callout'
+description: A Callout is a banner that highlights important information on a page.
+slug: /components/callout/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Stack} from '@twilio-paste/stack';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Callout, CalloutHeading, CalloutText, CalloutList, CalloutListItem} from '@twilio-paste/callout';
+import Changelog from '@twilio-paste/callout/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/callout"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/callout/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Callout"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/callout/"
+  storybookUrl="/?path=/story/components-callout--variants"
+  description={props.data.mdx.frontmatter.description}
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{Callout, CalloutHeading, CalloutText}} language="jsx">
+  {`<Callout variant='neutral'>
+    <CalloutHeading as='h3'>Heads up!</CalloutHeading>
+    <CalloutText>This is some information you need to know.</CalloutText>
+</Callout>`}
+</LivePreview>
+
+## Guidelines
+
+### About Callout
+
+Use a Callout to highlight important information in a page of text. Callouts should not be used to provide feedback to a user or alert them to a change in the system.
+
+### Accessibility
+
+We recommend Callouts have a CalloutHeading so that the information is easier to navigate to for people who use screen readers. The CalloutHeading's level should be set appropriately with the `as` prop. It is important to not skip one or more heading levels as it is common for screen readers to build an outline of the page and assist users as they navigate a page.
+
+## Examples
+
+### Callout variants
+
```

Comment:
issue: I think this needs more information, especially the names of the variants, explaining that the icon is included in every variant, and that the icon changes depending on the variant. It also might be useful to restate that this is for information hierarchy purposes, not interactivity, and link to alert and/or toast for elements that provide feedback and interactivity.